### PR TITLE
Add docker based action for backporting a pull request

### PR DESCRIPTION
**Pull request by bjoernricks**
**What**:

Add a docker based action for backport a pull request via issue labels

**Why**:

Use own action to replace mergify

**How**:

**Checklist**:

- [ ] Tests
- [ ] CHANGELOG.md entry
- [ ] Documentation

### CODE REVIEW

File: backport-pull-request/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      - closed
+      - labeled
+
+jobs:
+  backport:
+    name: Backport Pull Requests
+    runs-on: ubuntu-latest
+    steps:
+        - name: Backport Pull Request
+          uses: bjoernricks/pull-request-backport-action@main
```
Comment (y0urself) [inline backport-pull-request/README.md:23]:
```suggestion
          uses: greenbone/actions/pull-request-backport-action@v1
```

File: backport-pull-request/README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+[backport.rule-1]
+label = "backport-to-main"
+source = "develop"
+destination = "main"
+
+[backport.rule-2]
+label = "backport-to-staging"
+source = "main"
+destination = "staging"
+```
```
Comment (y0urself) [inline backport-pull-request/README.md:80]: Would love to also already have an `example.toml` file, so users can start with copy-pasting that file.

File: backport-pull-request/README.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+## Action Configuration
+
+|Input Variable|Description| |
+|--------------|-----------|-|
+|token|Token required to create the backport pull request|Required|
+|config|TOML based configuration file for backporting pull requests|Optional (default is `backport.toml`)|
+
+## TOML Configuration
+
+The TOML configuration specifies when a pull request and where it should be
```
Comment (y0urself) [inline backport-pull-request/README.md:53]:
```suggestion
The TOML configuration specifies when and where a pull request should be
```

File: backport-pull-request/README.md (hunk 1 above)
Comment (bjoernricks) [reply backport-pull-request/README.md]: Should be `greenbone/actions/backport-pull-request@v1`. thanks

File: backport-pull-request/README.md (hunk 2 above)
Comment (bjoernricks) [reply backport-pull-request/README.md:80]: I've added a `backport.example.toml` file

**Review by y0urself — APPROVED** [review]

**Pull request merged**